alvr/xtask/src/packaging.rs

```diff
@@ -44,6 +44,8 @@ fn build_windows_installer() {
     .unwrap();
 }
 
+const DEFAULT_PROFILE: build::Profile = build::Profile::Distribution;
```

Comment:
We don't need to define a default profile :)

Reply:
It will help on noticing things depending on the profile, there is a ``copy_file`` based on the profile build dir on the windows code which previously said ``"release"``

Reply:
Making a const will prevent bugs like those being introduced